### DISCUSSION TITLE
Portfolio/Post/Gallery Widget Enhancements

### DISCUSSION THREAD

**Discussion by cranstantinople** [elementor/elementor#17347] [Developers Community]
The Portfolio, Post and gallery Widgets have a lot of overlapping features but each seem to be missing certain customizations to get exactly what I'm trying to accomplish.

The portfolio widget is the closest to what I'm looking for out of the box but it needs more customization options for the Title/Text.  I pull all the posts with a "Vendor" category but not all of our Vendors have a Logo/Photo so ideally, there would be an option to overlay the Title/Text on all the cards or only cards with no image so the Vendor Name would be visible even without a logo.

The portfolio widget also doesn't really have any options for adjusting card parameters such as padding, border, etc.

https://www.schattenbol.com/market/

**Comment by 20mcdonald607** [comment]
As above, I also find the image options between the the gallery and the portfolio should be more consistent.

I would also like to be able to combine images and videos in one gallery. We previously has a 10 year old theme that was able to do this. So it shouldn't be that difficult.
